src/About/about.css

```diff
@@ -0,0 +1,90 @@
+body{
+  background-image: url(./images/About\ Me.jpg);
+  background-repeat: no-repeat;
+  background-size: 1500px 800px;
+  color: #555f77;
+  background-attachment: fixed;
+  font-family: 'Roboto', sans-serif;
+}
+
+a{
+  border-bottom: 4px solid #e4dfdf;
+  text-decoration: none;
+  color: #555f77;
+  transition: border 0.3s;
+  padding-bottom: 22px;
+}
+
+a:hover {
+  border-bottom: 4px solid;
+}
+
+#header-container {
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  height: 46px;
+  z-index: 1050;
+  background-color: #e4dfdf;
+  box-shadow: 0px 2px 10px 0px black;
+  padding: 10px 20px;
+  overflow: hidden;
+}
+
+.header-fix {
+  height: 65px;
+}
+
+.main-container .content {
+  margin: 50px;
+  padding: 50px;
+  background: #e4dfdf;
+  border-radius: 10px;
+  box-shadow: 0px 2px 10px 0px black;
+  opacity: .55;
+  color: black;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  width: 500px;
```

Comment:
change the `width` to `max-width: 500px;`. at it's full width it'll 500px like you want. but in mobile it'll be whatever size it needs to be to fit the screen size.

Reply:
ah that makes sense!